UnitySDK/Assets/ML-Agents/Scripts/Sensor/WriteAdapter.cs

```diff
@@ -1,5 +1,6 @@
 using System.Collections.Generic;
 using MLAgents.InferenceBrain;
+using UnityEngine;
```

Comment:
I would rather not import UnityEngine unless necessary. Here it is for Debug.Assert, which I think we can replace with if, throw and else statements.

Reply:
Will do. For my own enlightenment, what's the problem with extra `using`s? Slower compile time, or just aesthetic objections?